### DISCUSSION TITLE
Suggestion about colors in https://tailwindcss.com

### DISCUSSION THREAD

**Discussion by 4lejandrito** [tailwindlabs/tailwindcss#2811] [General]
Hi!

I've been reading through the new site and trying out the examples and some of them don't work out of the box. The examples that I found not working are all those using the `teal` or `light-blue` colors (gradients for example).

<img width="1400" alt="Screenshot 2020-11-19 at 10 53 53" src="https://user-images.githubusercontent.com/1937354/99650296-8660a700-2a55-11eb-9e40-a0960c101cc6.png">

I was very confused and I went to https://play.tailwindcss.com to see if they worked there and some of them did. Then I found that you are using some custom config in the config tab:

<img width="640" alt="Screenshot 2020-11-19 at 10 49 28" src="https://user-images.githubusercontent.com/1937354/99649775-e9057300-2a54-11eb-9b63-ad44dfcf7c53.png">

I added the `teal` and `light-blue` colors to my config file and then everything was fine.

I suggest you don't use custom configuration in https://tailwindcss.com because users are going to be confused when they try out the examples in their own sites. Either use only out of the box stuff or clearly specify which examples need extra config.

Thanks for making Tailwind, I love it.

**Comment by deadcoder0904** [comment]
Yes, I also thought the same. However, I want to use `blueGray` & am confused whether they are included by default or we have to extend them in config.

It doesn't help to know what version Tailwind Intellisense supports.

**Edit: Just added `blueGray` in the config & it started working. Even Intellisense picks up properly 😍**

**Comment by adamwathan** [comment]
Hey! Hopefully the customizing colors documentation is helpful here, it explains all this:

https://tailwindcss.com/docs/customizing-colors

We are careful to use only default colors throughout the actual documentation examples but for the homepage we needed more to achieve what we wanted to achieve visually.

**Comment by 4lejandrito** [comment]
I see your point and it makes sense. However one of the things that I love about Tailwind is that is so easy to copy the classes from your examples and apply them to my HTML. For a few minutes I thought there was a bug or I hadn't installed Tailwind@2 properly. 

At the end I could achieve what I wanted and the docs were indeed really helpful, it just took me a while.

Feel free to mark this as resolved.

Thanks!
